doc/pages/administrator/040-manual_kubernetes.md

```diff
@@ -1,73 +1,55 @@
+In case you do not want to use `helm` to manage the Operator, this guide will run you through
+all the steps needed to set up Astarte Kubernetes.
+
+To come along with this guide, the following components are required:
++ [`operator-sdk`](https://sdk.operatorframework.io/docs/installation/install-operator-sdk/)
++ [`kustomize`](https://kubectl.docs.kubernetes.io/installation/kustomize/)
+
+Please make sure that the version of `operator-sdk` matches or is compatible with the version
+of the `operator-sdk` module in the Operator's `go.mod` file.
 
 *Note: Please be aware that this method is to be used only if you have very specific reasons why not to
-use `astartectl`, for example: you're running a fork of the Operator, you're running the Operator
+use `helm`, for example: you're running a fork of the Operator, you're running the Operator
 outside of the cluster, or you're on the very bleeding edge.
-`astartectl` automates internally all of this guide and should be your main choice in production.*
+`helm` automates internally all of this guide and should be your main choice in production.*
 
 ## Clone the Operator Repository
 
 First of all, you will need to clone the Operator repository, as this is where some of the needed resources
-for the Operator are. Ensure you're cloning the right branch for the Operator Version you'd like to install.
-For example, if you want to Deploy an Operator in the 0.11 series, you would
+for the Operator are. Ensure you're cloning the right branch for the Operator Version you'd like to install,
+in this case v1.0:
 
 ```bash
 git clone https://github.com/astarte-platform/astarte-kubernetes-operator.git
+cd astarte-kubernetes-operator
+git checkout release-1.0
 ```
 
 ## Install RBACs and CRDs
 
 The Operator requires a number of RBAC roles to run, and will also require Astarte CRDs to be installed.
 
-Navigate into the `deploy` directory of your local clone, and install the service account:
-
+To install all the required components, simply run:
 ```bash
-kubectl apply -f service_account.yaml
-kubectl get ServiceAccount -n kube-system astarte-operator
+make install
 ```
 
-Then, install the Cluster Role
+## Running the Operator inside the Cluster
 
+Running the Operator inside the cluster is as simple as executing the following:
 ```bash
-kubectl apply -f role.yaml
-kubectl get ClusterRole astarte-operator
+make deploy
 ```
 
-Last but not least, install the Cluster Role Binding:
+Actually, the above command does more than just deploying the Operator, as it also install RBACs, CRDs. The deployment
+therefore can be performed in just one command.
 
-```bash
-kubectl apply -f role_binding.yaml
-kubectl get ClusterRoleBinding astarte-operator
-```
-
-Once done, navigate into the `deploy/crds` directory of your local clone, and install all Astarte Custom
-Resource Definitions:
+Be aware that `make deploy` shares the same `kubectl` Kubernetes context.
```

Comment:
I'd move this at the beginning of the Manual installation section, something like "Make sure that the cluster `kubectl` is pointing to is the one you want to target with the installation"